reports/pin/pin.qmd

```diff
@@ -15,23 +15,364 @@ format:
     fontsize: 12pt
 editor: source
 params:
-  run_id: "2024-01-10-objective-patrick"
+  run_id: "2024-01-07-great-ida"
   year: "2024"
-  pin: "13253180150000"
+  pin:
+    - "16083110070000"
 ---
 
 {{< include ../_setup.qmd >}}
 
-```{r pin_checks}
-target_pin <- ccao::pin_clean(params$pin)
+```{r _pin_checks}
+target_pins <- ccao::pin_clean(params$pin)
 
-if (nchar(target_pin) != 14) {
-  stop("Invalid PIN: must be 14 characters long")
+for (target_pin in target_pins) {
+  if (nchar(target_pin) != 14) {
+    stop(paste("Invalid PIN:", target_pin, "must be 14 characters long"))
+  }
+
+  if (!target_pin %in% assessment_card$meta_pin) {
+    stop(paste("Invalid PIN:", target_pin, "not found in assessment data"))
+  }
 }
+```
 
-if (!target_pin %in% assessment_card$meta_pin) {
-  stop("Invalid PIN: not found in assessment data")
+```{r _pin_URLs, results='asis'}
+base_url <- "https://www.cookcountyassessor.com/pin/"
+pin_values <- params$pin
+
+for (pin in pin_values) {
+  full_url <- paste0(base_url, pin)
+  cat(sprintf('<a href="%s">%s</a><br/>', full_url, full_url))
```

Comment:
**issue (blocking):** Likewise for the rest of this report: this report always targets a single PIN, so no need for loops.